src/Rings.jl

```diff
@@ -241,3 +241,40 @@ function is_finite(R::NCRing)
   c == 1 && return true
   throw(NotImplementedError(:is_finite, R))
 end
+
+function _is_noetherian(R::T) where T<:Ring
+  if R isa Union{PolyRing, MPolyRing, LaurentPolyRing, LaurentMPolyRing}
+    return _is_noetherian(coefficient_ring(R))
+  elseif R isa Union{MSeriesRing, SeriesRing}
+    return _is_noetherian(base_ring(R))
```

Comment:
I was about to complain about this, but it actually seems OK for now -- but it made me write issue #2079 :-)